ocaml/xapi/create_misc.ml

```diff
@@ -109,28 +112,12 @@ let read_dom0_memory_usage () =
   with _ ->
     None
 
-let read_localhost_info () =
-  let xen_verstring, total_memory_mib =
-    try
-      let xc = Xenctrl.interface_open () in
-      let v = Xenctrl.version xc in
-      Xenctrl.interface_close xc;
-      let open Xenctrl in
-      let xen_verstring = Printf.sprintf "%d.%d%s" v.major v.minor v.extra in
-      let total_memory_mib =
-        let open Xapi_xenops_queue in
-        let module Client = (val make_client (default_xenopsd ()) : XENOPS) in
-        Client.HOST.get_total_memory_mib "read_localhost_info" in
-      xen_verstring, total_memory_mib
-    with e ->
-      if Pool_role.is_unit_test ()
```

Comment:
Do the unit tests pass now, or do you need to add some special handling in the unit test to mock out the API call?

Reply:
I didn't mock any API call to make the unit tests pass neither on Travis nor on my localhost.

Reply:
`read_localhost_info` does not seem to be used from a unit test right now, so the exception handler is probably not useful for that. The other part  of the exception handler looks like it is meant to cover the case where xapi is not running on a Xen system, which isn't something that is relevant anymore.